<ui version="4.0" >
 <class>TTCutOutFrameWidget</class>
 <widget class="QWidget" name="TTCutOutFrameWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>2</horstretch>
    <verstretch>8</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>280</width>
    <height>260</height>
   </size>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="gbCutOutFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="0" column="0" >
       <widget class="TTMPEG2Window2" name="mpegWindow" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>2</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>4</number>
        </property>
        <property name="sizeConstraint" >
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="laCutOutFramePosition" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string>00:00:00,000 (000000)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPrevCutOutFrame" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="cutoutframewidget.qrc" >
            <normaloff>:/pixmaps/pixmaps/prev_18.xpm</normaloff>:/pixmaps/pixmaps/prev_18.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbNextCutOutFrame" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="cutoutframewidget.qrc" >
            <normaloff>:/pixmaps/pixmaps/next_18.xpm</normaloff>:/pixmaps/pixmaps/next_18.xpm</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSearchFrame" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
          <property name="text" >
           <string>Search</string>
          </property>
          <property name="icon" >
           <iconset resource="cutoutframewidget.qrc" >
            <normaloff>:/pixmaps/pixmaps/search_18.xpm</normaloff>:/pixmaps/pixmaps/search_18.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTMPEG2Window2</class>
   <extends>QFrame</extends>
   <header>../mpeg2window/ttmpeg2window2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cutoutframewidget.qrc" />
 </resources>
 <connections/>
</ui>
